observations/models.py

```diff
@@ -76,21 +77,25 @@ class Observation(PolymorphicModel):
     """
     value = models.ForeignKey(
         AllowedValue, blank=False, null=True,
+        on_delete=models.CASCADE,
```

Comment:
Here, it seems that corresponding observations are deleted if an allowed value is deleted. Perhaps we want to make sure that no observations use a value before deleting it?

Reply:
Observation deletion prevented in 35306b441fac9c4352c57a3873c114a2e1912663
Actual soft deletion added to backlog.